Sources/StreamChatUI/MessageActionsPopup/ChatMessagePopupVC.swift

```diff
@@ -60,14 +62,17 @@ open class ChatMessagePopupVC: _ViewController, ComponentsProvider {
         view.embed(blurView)
 
         messageContainerStackView.axis = .vertical
-        messageContainerStackView.spacing = 8
+        messageContainerStackView.spacing = 0
         view.addSubview(messageContainerStackView)
 
         var constraints: [NSLayoutConstraint] = [
-            messageContainerStackView.leadingAnchor.pin(greaterThanOrEqualTo: view.leadingAnchor),
-            messageContainerStackView.trailingAnchor.pin(lessThanOrEqualTo: view.trailingAnchor),
             messageContainerStackView.bottomAnchor.pin(lessThanOrEqualTo: view.bottomAnchor),
-            messageContainerStackView.topAnchor.pin(greaterThanOrEqualTo: view.topAnchor)
+            messageContainerStackView.topAnchor.pin(greaterThanOrEqualTo: view.topAnchor),
+            messageContainerStackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0),
+            messageContainerStackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0),
+            // Keeping old constraint for future reference
```

Comment:
@jitendra-simform can we remove commented code? if we do required old constaints info we can get from older commit.